rasa/nlu/utils/spacy_utils.py

```diff
@@ -39,6 +38,24 @@ def __init__(
         self.nlp = nlp
         super(SpacyNLP, self).__init__(component_config)
 
+    @staticmethod
+    def load_model(spacy_model_name):
+        """Try loading the model, catching the OSError if missing."""
+        import spacy
+
+        try:
+            nlp = spacy.load(spacy_model_name, disable=["parser"])
```

Comment:
No need to store the result of `spacy.load` into a variable. You can directly return it.